public/css/jumbotron-narrow.css

```diff
@@ -74,6 +73,29 @@ body {
   margin-top: -75px;
 }
 
+.campaign-form {
+  font-size: 16px;
+  background-color: white;
+  color: #35A5B3;
+  margin-left: 50px;
+  margin-right: 70px;
```

Comment:
Something's screwy with the right side of the app view, it's always farther than the left